resources/logging/templates/loki/deployment.yaml

```diff
@@ -1,3 +1,4 @@
+{{/*
```

Comment:
this file can be removed completely, it's replaced with statefulset.yaml

Reply:
Fixed